### ISSUE TITLE
mkdocs admonition math formula

### ISSUE THREAD

**Issue by dodoni** [mbarkhau/markdown-katex#8]
There seems to be a problem with display mode formulas: ```math inside an admonition (https://squidfunk.github.io/mkdocs-material/reference/admonitions/).  For example

```
!!! hint 
    It follows a simple formula not shown correctly:

    ```math
    \begin{gather}
     sin(x)
    \end{gather}
     ```
```
Shows the code instead the formula. Without indentation the formula is shown, but not inside the admonition.

**Comment by mbarkhau** [comment]
Fair point. I would add this is probably also broken for blockquotes.

I think the place to fix this is in [markdown_katex/extension.py@_iter_out_lines](https://github.com/mbarkhau/markdown-katex/blob/master/src/markdown_katex/extension.py#L216).

Let me know if you're interested on working on this.

**Comment by mbarkhau** [comment]
Fixed in [`202106.1032`](https://pypi.org/project/markdown-katex/202106.1032/)
